scripts/release/.gitignore

```diff
@@ -1,2 +1,2 @@
 .env
-Rplots.pdf
\ No newline at end of file
+Rplots.pdf
```

Comment:
Guess you can add all R / RStudio related files to .ignore here

Reply:
I think this is the only one I need to ignore